controls/radrichtextbox/features/selection.md

```diff
@@ -265,6 +265,10 @@ Here is an example of how you can use the first approach.
 ![Selection in RadRichTextBox](images/RadRichTextBox_Selection_01.png)
 
 
+## Other Selection Properties
+
+*  __MouseSelectionHandler.DoubleClickTime__: This static property controls the double click speed of the RichTextBox.

```

Comment:
- API should be highlighted with the " ` " character, instead of bolding
- Instead of " : ", use the " — "

Reply:
Fixed